easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.2.3-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'AUGUSTUS'
+version = '3.2.3'
+versionsuffix = '-Python-2.7.13'
+
+homepage = 'http://bioinf.uni-greifswald.de/augustus/'
+description = "AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences"
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://bioinf.uni-greifswald.de/augustus/binaries/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['AUGUSTUS-%(version)s_fix-hardcoding.patch']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Boost', '1.63.0', versionsuffix),
+    ('SQLite', '3.17.0'),
+    ('GSL', '2.3'),
+    ('SuiteSparse', '4.5.5', '-ParMETIS-4.0.3'),
+    ('lpsolve', '5.5.2.5'),
+    ('BamTools', '2.4.1'),
+]
+
+skipsteps = ['configure']
+
+prebuildopts = "unset LDFLAGS && unset LIBS && "
```

Comment:
Having these defined results in overruling the values set by AUGUSTUS itself, causing the build to fail.

There's nothing in here that's really required, libraries will still be picked up correctly via `$LIBRARY_PATH`.